packages/flutter_tools/gradle/build.gradle.kts

```diff
@@ -32,3 +50,7 @@ dependencies {
     //  * AGP version in buildscript block in packages/flutter_tools/gradle/src/main/flutter.groovy
     compileOnly("com.android.tools.build:gradle:7.3.0")
 }
+
+tasks.named<Jar>("jar") {
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
```

Comment:
Yup I agree with all of that. This is a temporary hack because for some reason, I started getting duplicate class error on `FlutterPlugin`.

Will debug and once I find a solution, I'll get rid of this.